### PR TITLE
feat(android): track color of the Ti.UI.Switch

### PR DESCRIPTION
**Pull request by m1ga**
Parity for `tintColor` and adding `onTintColor` to color the tracks for the Material Switch.
Also adds `thumbColor` and `onThumbColor`.

```js
var win = Ti.UI.createWindow();

var basicSwitch = Ti.UI.createSwitch({
  tintColor: "red",
  onTintColor: "green",
  onThumbColor: "yellow",
  thumbColor: "blue"
});

// basicSwitch.tintColor = "blue"
win.add(basicSwitch);
win.open();

```

Screenshot_20240603-140140
Screenshot_20240603-140145

with the thumbColor:
Screenshot_20240604-142455

**Current workaround**
Using an [Android theme](https://m2.material.io/develop/android/components/switches#theming-switches)

### CODE REVIEW

**Comment by hansemannn** [comment]
IIRC, we have similar theming capabilities on iOS already, are these the same properties?

**Comment by m1ga** [comment]
ah, good point. `onTintColor` is the color for "on". I'll rename the `activeTintColor`

**Comment by m1ga** [comment]
Update: 
* renamed `activeTintColor` to `onTintColor` for iOS parity.
* renamed `activeThumbColor` to `onThubColor` to match the other property

File: apidoc/Titanium/UI/Switch.yml [hunk 1]
```diff
@@ -118,17 +118,43 @@ properties:
...
+
+        Android: Active thumb color of the Material Switch.
+    type: [String, Titanium.UI.Color]
+    default: undefined
+    platforms: [android]
+    since: {android: 12.4.0}
+
+  - name: thumbColor
+    summary: The color used to tint the thumb icon of the switch when it is turned off.
+    description: |
```
Comment (hansemannn) [inline apidoc/Titanium/UI/Switch.yml:143]: If no specific description is provided, you can rather onit the field to prevent duplicate content

Comment (m1ga) [reply apidoc/Titanium/UI/Switch.yml:141]: I removed the summery so it will just use the description as that had more info

**Pull request merged**